doc/source/releases/21.01_announce.rst

```diff
@@ -1,11 +1,82 @@
+  and improved job creation performance for certain classes of tools and copying Histories, Datasets and Dataset Collections (`Pull Request 10539`_, `Pull Request 10549`_).
+  The performance of reading tool wrappers when starting Galaxy has been much improved. We have therefore decided to disable the tool document cache by default,
+  since it only provides negligible speedups (`Pull Request 10804`_, `Pull Request 11447`_). The tool document cache can be re-enabled with the `enable_tool_document_cache <https://github.com/galaxyproject/galaxy/blob/283ec1772fcc0e574997dc2c5ad12dc86e3e1d37/lib/galaxy/config/sample/galaxy.yml.sample#L626>`_ setting.
+
+**Offload Zip Archive Creation to NGINX**
+  Galaxy can now utilize the NGINX module `mod_zip <https://www.nginx.com/resources/wiki/modules/zip/>`__ to assemble and compress dataset collection and library dataset archives on the fly.
+  This method is more efficient and can create large archives without affecting the Galaxy server process. The required setup is described in the admin documentation for `proxying Galaxy with NGINX <https://docs.galaxyproject.org/en/release_21.01/admin/nginx.html#creating-archives-with-mod-zip>`__.
+
+**Workflow Enhancements and Fixes**
+  This release contains many new features, enhancements and bug fixes for creating, running and reporting workflows described in greater details in the `21.01 user release notes <21.01_announce_user.html>`__
+
+**History Import and Export**
+  Histories can now be imported from and exported to the pluggable file source architecture introduced in the 20.09 release of Galaxy.
+  This includes the users' FTP folder, S3, dropbox, owncloud etc. In addition the History Export and Import dialogues have been rewritten
+  to provide access to these new options and to provide more feedback during Import and Export process.
+
+Also check out the `21.01 user release notes <21.01_announce_user.html>`__
+
+Deprecation Notices
+===========================================================
+
+**Deprecation of support for Python 3.5**
+  Galaxy requires Python 3.6 or newer. Instruction for updating Python can be found in the `Supported Python versions <https://docs.galaxyproject.org/en/release_21.01/admin/python.html#supported-python-versions>`__ section of the documentation.
+  Note that if you update to Python 3.7.2 or newer you need to set py-call-osafterfork to ``false`` in the uwsgi section of galaxy.yml.
+
+**Removal of IUC conda channel from default configuration**
+  The IUC conda channel has been removed from the default configuration. This channel was used for installing dependencies that were old and not available in bioconda or conda-forge. These dependencies are now available in Bioconda. By removing this channel Conda can work more efficiently and ignore outdated dependencies.
```

Comment:
```suggestion
  The IUC conda channel has been removed from the default configuration. This channel was used for installing dependencies that were old and not available in bioconda or conda-forge. These dependencies are now available in bioconda. By removing this channel Conda can work more efficiently and ignore outdated dependencies.
```
I think in both cases "bioconda" should be lowercased as it refers to the name of a channel. I think uppercasing "Conda" in the second sentence is correct.